README.md

```diff
@@ -60,50 +44,65 @@ Alternately, you can install the plug-in from the Eclipse Marketplace.
-* Creates a `slice` subdirectory to contain your Slice files. The plug-in automatically compiles any Slice file that you add to this directory.
-* Creates a `generated` subdirectory to hold the Java source files that the slice2java translator generates from your Slice files.
-* Adds a library reference to the Ice runtime JAR file (`Ice.jar`). The plug-in assumes that the JAR file resides in the `lib` subdirectory of your Ice installation.
+The plug-in is enabled on a per project basis. To enable it, right click on the project, choose `Ice Builder` and select `Add Ice Builder`.
+
+Upon activation, the project immediately creates a `generated` directory to hold the Java source files the Slice compiler generates from your Slice files.
+![Activating the Plug-in](/Screenshots/activation.png)
+To de-activate the plugin, simply navigate to the `Ice Builder` menu and instead select `Remove Ice Builder`. Note that removing the Ice Builder plug-in
+will not affect your Slice files, but will remove all generated code.
 
 ### Configuring Project Settings
 
-To configure the project-specific settings, select `Properties` from the `Project` menu or right-click on the name of your project and choose `Properties`. Click on `Ice Builder` to view the plug-in's configuration settings, which are presented in two tabs: Source and Options.
+To configure the project-specific settings, select `Properties` from the `Project` menu or right-click on the name of your project and choose `Properties`.
+Click on the `Ice Builder` menu to to view the plug-in's settings for the selected project.
+![Project Properties Dialog](/Screenshots/properties.png)
+The `Generated Code Directory` allows you to specify where all generated code should be placed in the project. By default, the plug-in uses the `generated`
+directory for this. To change the directory, you must first create the new directory, and then click `Browse` to navigate to it. The new directory must be empty,
+and the plug-in requires exclusive use of it; any files or changes made to the directory will be lost when the Slice compiler is run.
 
-#### Settings in the Source Tab
+Any external Slice files your project is dependent upon can be specified using the `Include Directories` interface, where you can add directories 
+the compiler can search for referenced Slice definitions. These directories are searched in the order they're listed in the interface.
 
-This tab configures the directories of your Slice files and generated code. The plug-in includes the `slice` subdirectory by default, but you can remove this directory and add other directories if necessary. The plug-in only compiles Slice files that are located in the configured subdirectories.
+For more advanced users, additional options can be passed directly to the Slice compiler with the `Advanced Options` section. For a list of all supported
```

Comment:
The project properties dialog currently calls this "Additional Options", not "Advanced Options"